pages/price-feeds/use-real-time-data/evm.mdx

```diff
@@ -1,10 +1,16 @@
----
-description: Consume Pyth Network prices in EVM applications
----
+import { Callout } from "nextra/components";
 
-# How to Use Real-Time Data in EVM Contracts
+# How to Pull Real-Time Data in EVM Contracts
 
-This guide explains how to use real-time Pyth data in EVM contracts. For an interactive playground to explore the methods supported by the Pyth contract, see the [EVM API reference](../api-reference/evm/).
+This guide explains how to **Pull real-time Pyth** data in EVM contracts. For an interactive playground to explore the methods supported by the Pyth contract, see the [EVM API reference](../api-reference/evm/).
+
+<Callout type="info" emoji="ℹ️">
```

Comment:
I believe we should start with this simple callout in every guide. 
If we all feel good with this, then I will add this to every ecosystem guide.

Reply:
Using push is simple and straightforward, and it will prevent experienced folks to go to another page. In my opinion, it is giving just enough information to start consuming the price in one page.